verticapy/tests/vModel/test_lasso.py

```diff
@@ -147,10 +150,15 @@ def test_to_sklearn(self, model):
         prediction = model.cursor.fetchone()[0]
         assert prediction == pytest.approx(md.predict([[3.0, 11.0, 93.0]])[0][0])
 
-    @pytest.mark.skip(reason="shap doesn't want to work on python3.6")
-    def test_shapExplainer(self, model):
-        explainer = model.shapExplainer()
-        assert explainer.expected_value[0] == pytest.approx(5.81837771)
+    try:
+        import shap
```

Comment:
I think we skipped this test because adding the `shap` package to the deps list in tox.ini was causing failure under Py3.6 in the previous test system. Now that it is successfully added, you should be able to run this test normally without any limit.
Btw, you can skip conditionally using the `skipif` marker:
https://docs.pytest.org/en/latest/skipping.html#skipif